PolicyServicePkg/Library/DxePolicyLib/DxePolicyLib.inf

```diff
@@ -31,6 +31,7 @@
   BaseLib
   DebugLib
   UefiBootServicesTableLib
+  MemoryAllocationLib

```

Comment:
You could maintain sorted order in these which is nice visually if you moved the `MemoryAllocationLib` addition in place.

```suggestion
  BaseLib
  DebugLib
  MemoryAllocationLib
  UefiBootServicesTableLib


```